src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed === true,
+  })}
+  >
+    <h2 className="TodoInfo__title">
+      { todo.title }
+    </h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
the `getUserById` function may return `null`, i.e. `todo.user` might be null theoretically
you should handle this case (do not render the UserInfo component at all in this case) - use conditional rendering